cinema/urls.py

```diff
@@ -1 +1,29 @@
-# write urls here
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from cinema.views import (
+    GenreViewSet,
+    MovieViewSet,
+    ActorViewSet,
+    MovieSessionViewSet,
+    CinemaHallViewSet,
+)
+
+app_name = "cinema"
+router = DefaultRouter()
+router.register("genres", viewset=GenreViewSet, basename="genres")
+router.register("movies", viewset=MovieViewSet, basename="movies")
+router.register("actors", viewset=ActorViewSet, basename="actors")
+router.register(
+    "cinema_halls",
+    viewset=CinemaHallViewSet,
+    basename="cinema_halls"
+)
+router.register(
+    "movie_sessions",
+    viewset=MovieSessionViewSet,
+    basename="movie_sessions"
+)
+urlpatterns = [
+    path("", include(router.urls)),
+    path("app/cinema/", include(router.urls)),
```

Comment:
The `urlpatterns` include the router URLs twice. Remove one of these path inclusions to avoid duplicate routes and potential confusion.